examples/mundialis_example.py

```diff
@@ -28,11 +36,12 @@
 
 # Test Processes
 # North Carolina MODIS LST example
-datacube = con.imagecollection("nc_spm_08.modis_lst.strds.MOD11B3.A2016306.h11v05.single_LST_Day_6km")
-datacube = datacube.filter_bbox(west=-80.5, south=32.5, east=-70.0, north=38.6, crs="EPSG:4326")
-datacube = datacube.filter_temporal(extent=["2016-01-01T00:00:00Z", "2016-03-10T23:59:59Z"])
+datacube = con.imagecollection("utm32n.openeo_bolzano.strds.openeo_bolzano_S2")
```

Comment:
FYI: instead of `con.imagecollection`  there is also `con.load_collection()`, which is more in line with the openEO API processes